recipes/augustus/3.2.2/meta.yaml

```diff
@@ -0,0 +1,119 @@
+about:
+  home: http://bioinf.uni-greifswald.de/augustus/
+  license: ''
+  summary: 'AUGUSTUS is a gene prediction program for eukaryotes written by Mario
+    Stanke and Oliver Keller. It can be used as an ab initio program, which means
+    it bases its prediction purely on the sequence. AUGUSTUS may also incorporate
+    hints on the gene structure coming from extrinsic sources such as EST, MS/MS,
+    protein alignments and synthenic genomic alignments.'
+build:
+  number: 0
+package:
+  name: augustus
+  version: '3.2.2'
+  skip: True # [osx]
+requirements:
+  build:
+      - libgcc
+      - gcc
+      - bamtools
+      - perl-app-cpanminus
+      - perl-yaml
+      - perl-dbi
+      - boost
```

Comment:
So I'm having trouble getting it compiled without the boost libs.

I made the changes to the meta.yaml as in salmon.

This is the output from building on the bioconda image.
`+ make
In file included from genbank.cc:15:0:
../include/genbank.hh:26:48: fatal error: boost/iostreams/filtering_stream.hpp: No such file or directory
 #include <boost/iostreams/filtering_stream.hpp>
`

If I don't include these two lines in my build.sh:

export CXXFLAGS="-DUSE_BOOST -I${BOOST_INCLUDE_DIR} -L${BOOST_LIBRARY_DIR}"
export LDFLAGS="-L${BOOST_LIBRARY_DIR}"